File: Modul-01/c/PRAK106-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    float a, b, c;

    // Menggunakan scanf untuk menerima input dari pengguna
    printf("Variabel a bernilai ");
    scanf("%f", &a);
    
    printf("Variabel b bernilai ");
    scanf("%f", &b);
    
    printf("Variabel c bernilai ");
    scanf("%f", &c);

    // Melakukan perbandingan dan mencetak hasilnya
    int a_sama_dengan_b = (a == b);
    int b_lebih_besar_dari_c = (b > c);
    int a_tidak_sama_dengan_c = (a != c);

    printf("Apakah a sama dengan b ? jawabannya adalah %d\n", a_sama_dengan_b);
    printf("Apakah b lebih besar dari c ? jawabannya adalah %d\n", b_lebih_besar_dari_c);
    printf("Apakah a tidak sama dengan c ? jawabannya adalah %d\n", a_tidak_sama_dengan_c);

    return 0;
}

File: Modul-01/python/PRAK106-2310817210009-MuhammadIbnuSina.py
# Memasukkan Input nilai a, b, dan c
a = float(input("Masukkan nilai a: "))
b = float(input("Masukkan nilai b: "))
c = float(input("Masukkan nilai c: "))

# Memeriksa apakah a sama dengan b
if a == b:
    hasil1 = 1
else:
    hasil1 = 0

# Memeriksa apakah b lebih besar dari c
if b > c:
    hasil2 = 1
else:
    hasil2 = 0

# Memeriksa apakah a tidak sama dengan c
if a != c:
    hasil3 = 1
else:
    hasil3 = 0

# Menampilkan hasil
print(f"Apakah a sama dengan b ? jawabannya adalah {hasil1}")
print(f"Apakah b lebih besar dari c ? jawabannya adalah {hasil2}")
print(f"Apakah a tidak sama dengan c ? jawabannya adalah {hasil3}")
